bundles/net.tourbook.common/src/net/tourbook/common/Messages.java

```diff
@@ -134,6 +134,8 @@
    public static String        Graph_Label_Power_Unit;
    public static String        Graph_Label_Speed;
    public static String        Graph_Label_Temperature;
+   //CUSTOM TRACKS
```

Comment:
No need for the comment as the variable name is obvious. It also reduces the code "noise"